modules/ttdBidAdapter.js

```diff
@@ -58,7 +58,9 @@ function getBidFloor(bid) {
 }
 
 function getSource(validBidRequests) {
-  let source = {};
+  let source = {
+    tid: utils.deepAccess(validBidRequests[0], 'ortb2Imp.ext.tid')
```

Comment:
you had source.tid fine; but what I was imagining is that you would want something like 

https://github.com/prebid/Prebid.js/blob/d46661a4aa8994d57f3bea3055bd4e4c8aa50e10/modules/ixBidAdapter.js#L248 / https://github.com/prebid/Prebid.js/blob/d46661a4aa8994d57f3bea3055bd4e4c8aa50e10/modules/ixBidAdapter.js#L361

as you guys are asking other exchanges for that in other forums

Reply:
I commented below on where it would go

Reply:
Thanks I think I understand what you are getting at now.